index.js

```diff
@@ -32,7 +32,10 @@ module.exports = function staticCache(dir, options, files) {
   if (options.alias) {
     Object.keys(options.alias).forEach(function (key) {
       var value = options.alias[key]
-
+      if(process.platform=='win32'){
+        value=value.replace(/\//g,'\\')
+        key=key.replace(/\//g,'\\')
```

Comment:
> path.normalize coundn't convert \ to /

yes, but you are converting `/` to `\` in `win os`,
and `normalize` can handle this no need to check `os` by you self.

https://github.com/koajs/static-cache/blob/master/index.js#L50